netty/src/main/java/io/grpc/transport/netty/NettyClientTransport.java

```diff
@@ -192,8 +192,10 @@ public void shutdown() {
   }
 
   private void notifyShutdown(Throwable t) {
+    Status s = Status.UNAVAILABLE.withDescription("Transport is shutting down");
```

Comment:
When `t` is `null` it seems the Status should be OK. I think `t` is `null` when requested to shutdown (line 187) and when  the connection is closed gracefully (line 180).